src/services/authentication.service.ts

```diff
@@ -12,3 +13,15 @@ export const setName = async (
 export const isInMemberDatabase = (supabaseClient: SupabaseClient<Database>, id: string) => {
   return supabaseClient.from("member").select("*", { count: "exact", head: true }).eq("id", id)
 }
+
+export const resetPasswordForEmail = async (supabase: SupabaseClient, email: string) => {
+  const { error } = await supabase.auth.resetPasswordForEmail(email, {
+    redirectTo: `${window.location.origin}/api/auth/callback?next=/forgot-password/reset&`,
+  })
+  if (error) {
+    toast.error(error.message)
+    return false
+  } else {
```

Comment:
you technically don't need to wrap the `return true` in the else

Reply:
valid